appeals/web/src/server/lib/mappers/appeal/submappers/appellant-final-comment-due-date.mapper.js

```diff
@@ -11,15 +11,16 @@ export const mapAppellantFinalCommentDueDate = ({ appealDetails, currentRoute })
 			value: {
 				html:
 					// @ts-ignore
```

Comment:
I know this wasn't added in this PR, but it looks like this `ts-ignore` could be removed if `appellantFinalCommentDueDate` was added to the `WebAppealTimetable` type def in `appeals/web/src/server/appeals/appeal-details/appeal-details.types.d.ts`